1_wqp_pull.yml

```diff
@@ -9,88 +9,88 @@ packages:
   - scipiper
   - dplyr
   - xml2
-  
+
 sources:
   - 1_wqp_pull/src/wqp_inventory.R
   - 1_wqp_pull/src/wqp_pull.R
   - lib/src/require_local.R
-  
+
 targets:
   1_wqp_pull:
     depends:
       - 1_wqp_pull/out/wqp_data.rds.ind
-      
+
 # -- set pull date -- #
 # !! update pull date when you want to trigger a new inventory and pull
 # pull IDs will be generated with this date string YYYYMMDD
 
   wqp_pull_date:
-    command: c(I('20200723'))
-      
+    command: c(I('20210412'))
+
 # -- get a CONUS inventory of available data for download -- #
 
   wqp_pull_parameters:
     command: yaml.load_file('1_wqp_pull/cfg/wqp_pull_params.yml')
-    
+
   wqp_partition_cfg:
     command: yaml.load_file('1_wqp_pull/cfg/wqp_partition_config.yml')
-  
+
   # -- get inventory of observations available to download -- #
 
   # get inventory of samples available on WQP
   # only filters by characteristicNames, and pulls all at the same time
-  
-  states:
-    command: get_states(states_url = I('https://www.waterqualitydata.us/Codes/statecode?countrycode=US'))
-    
+
+  start_year:
+    command: c(I('1900'), I(seq('1980', '1997', by = 3)),
+      I(seq('1998', '2003', by = 2)), I(seq('2004', '2020')))
```

Comment:
Do these need to be characters? I think sequence is converting them to numbers.

Reply:
Yes, they need to be characters because task names takes a character input. Sequence isn't converting them to numbers, they  are being converted here [#L90-L95](url) using `as.numeric` .

Reply:
Seems the result of that `seq` is numeric already:
```r
class(seq('1998', '2003', by = 2))
[1] "numeric"
```

Reply:
Thank you Jordan for pointing this out, `class(seq('1998', '2003', by = 2))` is numeric and `start_year` is a character vector.

```
y <- c(('1900'), seq(1980, 1997, by = 3), seq(1998, 2003, by = 2), seq(2004, 2020))
> class(y)
[1] "character"
> class(y[2])
[1] "character"
> y
 [1] "1900" "1980" "1983" "1986" "1989" "1992" "1995" "1998" "2000" "2002" "2004" "2005" "2006" "2007"
[15] "2008" "2009" "2010" "2011" "2012" "2013" "2014" "2015" "2016" "2017" "2018" "2019" "2020"
```